tests/coalaCITest.py

```diff
@@ -172,3 +172,19 @@ def test_limit_files_negative(self):
             self.assertFalse(stderr)
             self.assertEqual(retval, 0,
                              'coala must return zero when successful')
+
+    def test_bear_dirs(self):
+        with bear_test_module(), \
+                prepare_file(['random_text'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                    coala.main, 'coala', '--non-interactive',
+                    '-c', os.devnull,
+                    '--bear-dir', 'test_bears',
+                    '-b', 'LineCountTestBear',
+                    '-f', filename,)
+            self.assertIn('This file has 1 lines.',
```

Comment:
The message in assert should be "This file has 1 **line** "

Reply:
Why?

Reply:
Oh yes. I am sorry i forgot about that.